### ISSUE TITLE
[Question or Bug] Text without spaces overflows cell width

### ISSUE THREAD

**Issue by Never7Dice** [simonbengtsson/jsPDF-AutoTable#687]
Space-less text overflows the width, like in this example:
![image](https://user-images.githubusercontent.com/34353798/86014863-8837b880-ba29-11ea-8424-fcb0a6561d9b.png)
If it was a bit longer it would actually break, but with this text "LONGUNSPLITTABLETEXTTHATHASNOSPACESATALL" it failed. It seems like the last letter can start printing or padding area.

Here's another example S letter overflowed to right padding area here also.
![image](https://user-images.githubusercontent.com/34353798/86015240-fa100200-ba29-11ea-9acd-b555bc9ae15f.png)

Style for those cells is pretty simple

```
          const paramCell: CellDef = {
            colSpan: 1,
            content: '\nLONGUNSPLITTABLETEXTTHATHASNOSPACESATALL'
            styles: { minCellWidth: this.minColumnWidth, cellPadding: this.cellPadding },
          };
```
The bold label is printed manually and is only represnted by `\n`

I think this is a bug in lib. Text should not overflow the padding area that much

**Comment by simonbengtsson** [comment]
Yes that looks like a bug! Can you post a complete example that reproduces the issue?

**Comment by Never7Dice** [comment]
I am using Angular 9 BTW

```
import * as jsPDF from 'jspdf';
import autoTable, { CellDef } from 'jspdf-autotable';
@Injectable({
  providedIn: 'root'
})
export class PdfTableExportService {
}
```

**Comment by mmghv** [comment]
Turned out to be a jspdf bug, reported here :
https://github.com/parallax/jsPDF/issues/3644
